ddtrace/tracer/option.go

```diff
@@ -52,6 +55,15 @@ func WithDebugMode(enabled bool) StartOption {
 	}
 }
 
+// NoErrorStack will disable the automatic generation of error stacks when setting the
+// error tag with a value of type error. However, it will still be possible to attach the
+// error stack manually, if desired, using the "error.stack" tag.
+func NoErrorStack() StartOption {
```

Comment:
Does this work as a StartSpanOption as well? If not, should we add one to be able to disable this behaviour per-span?

Reply:
No, this is a tracer start option and it will take effect mostly for integrations. Why would you want this for spans. For spans, you can just set a string as the error value if you don't want a stack trace. Were you thinking something else?

Reply:
You do have a point though. Given @niamster 's comment and yours here, this is making me reconsider this solution. Perhaps we need something different.

Reply:
I guess this might be a side effect of me liking and always using `span.Finish(WithError(err))`. If you want to use that idiom you can't disable the stack it seems (unless you disable it globally?).

Reply:
You're right. I've put back the `WIP` flag. Gonna give this more though.